packages/@react-native-windows/virtualized-list/src/VirtualizedList.js

```diff
@@ -1230,7 +1263,7 @@ class VirtualizedList extends React.PureComponent<Props, State> {
     this._fillRateHelper.computeBlankness(
       this.props,
       this.state,
-      this._scrollMetrics,
+      this._getScrollMetrics(),
```

Comment:
This smells a bit to me, to have cached scroll metrics be non-inverted, and have the accessor which inverts them. Easy to take in an upstream change which uses the raw variable, then have something acting differently.

Maybe we should update the cache in `componentDidUpdate` if inverstion state changes, so that the only state we have is the correct one.

Reply:
Indeed - I didn't spend a lot of time on this, but I was hoping to pull the scroll metrics and frame metrics out into a separate module that encapsulated these details so VirtualizedList could *only* get the scroll metrics via the accessor.